docs/networking/loadbalancer.md

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 4
+sidebar_label: Load Balancer
+title: "Load Balancer"
+keywords:
+- Load Balancer
+---
+_Available as of v1.2.0_
+
+Harvester load balancer is a built-in Layer 4 load balancer that can be used to distribute incoming traffic across workloads deployed on Harvester virtual machines (VMs) or guest Kubernetes clusters.
```

Comment:
```suggestion
The Harvester VM load balancer is a built-in Layer 4 load balancer that distributes incoming traffic across workloads deployed on Harvester virtual machines (VMs) or guest Kubernetes clusters.
```

Reply:
Harvester VM load balancer and Harvester load balancer are different.

Reply:
> Harvester VM load balancer and Harvester load balancer are different.

@yaocw2020 thanks for clarifying